src/js/index.js

```diff
@@ -0,0 +1,8 @@
+import ReactDOM from 'react-dom';
+import React from 'react';
+import Component from '~/Component';
```

Comment:
yup, that's what I mean.

I think this is good because, if we wanted to use this 'Component' file via multiple other files ( located in different directories ), the import would look different for each one ( as you noted above ).

But, by importing with reference to some constant location, the import path will be the same, which IMO is easier to maintain.